scripts/js.gradle

```diff
@@ -195,4 +191,5 @@ protoJs {
 
     generateParsersTask().dependsOn compileProtoToJs
     buildJs.dependsOn generateParsersTask()
+    testJs.dependsOn generateParsersTask()
```

Comment:
Let's discuss this one. 

I would make `testJs` dependent on `buildJs`. And rename `generateParsersTask` to `generateParsers`.

Reply:
`generateParsersTask()` is a part of 'io.spine.tools.proto-js-plugin'.